cargo-apk/src/manifest.rs

```diff
@@ -71,6 +73,9 @@ struct AndroidMetadata {
     /// Maps profiles to keystores
     #[serde(default)]
     signing: HashMap<String, Signing>,
+    /// Set up reverse port forwarding before the application launches
```

Comment:
```suggestion
    /// Set up reverse port forwarding before launching the application
```

Reply:
I don't understand the correction, both sentences describe the exact same thing in the exact same way?

Reply:
This way of wording makes it ever so slightly more clear that `cargo-apk` is (obviously) in control of launch_ing_ the application straight after. It is by all means a nit though, but I find it more clear.